Backpack/TappableLinkLabel/Classes/BPKTappableLinkLabel.m

```diff
@@ -92,14 +93,56 @@ - (void)resetLinks {
     [self.persistedLinks removeAllObjects];
 }
 
-- (void)updateTextColors {
-    self.contentView.linkAttributes =
-    @{NSForegroundColorAttributeName: self.linkColor, NSUnderlineStyleAttributeName: @(NSUnderlineStyleNone)};
+- (BPKFontStyle)getEmphasizedFontStyleFor:(BPKFontStyle)fontStyle {
+    switch (fontStyle) {
+    case BPKFontStyleTextCaps:
+        return BPKFontStyleTextCapsEmphasized;
+    case BPKFontStyleTextXs:
+        return BPKFontStyleTextXsEmphasized;
+    case BPKFontStyleTextSm:
+        return BPKFontStyleTextSmEmphasized;
+    case BPKFontStyleTextBase:
+        return BPKFontStyleTextBaseEmphasized;
+    case BPKFontStyleTextLg:
+        return BPKFontStyleTextLgEmphasized;
+    case BPKFontStyleTextXl:
+        return BPKFontStyleTextXlEmphasized;
+    case BPKFontStyleTextXxl:
+        return BPKFontStyleTextXxlEmphasized;
+    case BPKFontStyleTextXxxl:
+        return BPKFontStyleTextXxxlEmphasized;
+    case BPKFontStyleTextXlEmphasized:
+        return BPKFontStyleTextXlHeavy;
+    case BPKFontStyleTextXxlEmphasized:
+        return BPKFontStyleTextXxlHeavy;
+    case BPKFontStyleTextXxxlEmphasized:
+        return BPKFontStyleTextXxxlHeavy;
+    default:
+        NSAssert(
+            NO, @"Alternate style BPKTappableLinkLabels must have a more emphasized alternative to the fontStyle set.");
+    }
+    return fontStyle;
+}
 
-    self.contentView.activeLinkAttributes = @{
-                                        NSForegroundColorAttributeName: [self.linkColor colorWithAlphaComponent:0.2],
-                                        NSUnderlineStyleAttributeName: @(NSUnderlineStyleNone)
-                                        };
+- (void)updateTextColors {
+    UIColor *linkColor = self.style == BPKTappableLinkLabelStyleAlternate ? BPKColor.white : self.linkColor;
```

Comment:
Would it make sense to set some properties is `setStyle` instead of these `self.style == BPKTappableLinkLabelStyleAlternate` checks? Or create getters for these properties so checking the style should only happen at minimum places.